js/src/personal/manny/pages/About.module.css

```diff
@@ -0,0 +1,77 @@
+.container {
+  padding: 0 20px;
+  background-color: var(--mantine-color-dark-6);
+  color: var(--mantine-color-white);
+}
+
+.banner {
+  width: 100vw;
+  background-color: var(--mantine-color-dark-6);
+  text-align: center;
+  padding: 40px 0;
+  margin-left: calc(50% - 50vw);
+  border: 2px solid var(--mantine-color-white);
+  box-sizing: border-box;
+}
+
+.bannerTitle {
+  font-size: 5rem;
+  font-weight: bold;
+  color: var(--mantine-color-white);
+  margin: 0;
+}
+
+.buttonContainer {
+  display: flex;
+  justify-content: center;
+  margin-top: -1.5px;
```

Comment:
not the biggest fan of negative margins. What is this for? Is there another way of doing it that doesn't require negative margins?

Reply:
I used the negative padding to try to create the the illusion of the contact me button being attached to the banner but there probably is a better way to do this